2/contrib/openshift/bundle-plugins.txt

```diff
@@ -0,0 +1,118 @@
+# Generated file: To generate this file run 'make plugins-list' before pushing your changes
+ace-editor:1.1
+ant:1.11
+apache-httpcomponents-client-4-api:4.5.13-1.0
+authentication-tokens:1.4
+blueocean:1.24.8
+blueocean-autofavorite:1.2.4
+blueocean-bitbucket-pipeline:1.24.8
+blueocean-commons:1.24.8
+blueocean-config:1.24.8
+blueocean-core-js:1.24.8
+blueocean-dashboard:1.24.8
+blueocean-display-url:2.4.1
+blueocean-events:1.24.8
+blueocean-github-pipeline:1.24.8
+blueocean-git-pipeline:1.24.8
+blueocean-i18n:1.24.8
+blueocean-jwt:1.24.8
+blueocean-personalization:1.24.8
+blueocean-pipeline-api-impl:1.24.8
+blueocean-pipeline-editor:1.24.8
+blueocean-pipeline-scm-api:1.24.8
+blueocean-rest:1.24.8
+blueocean-rest-impl:1.24.8
+blueocean-web:1.24.8
+bootstrap4-api:4.6.0-3
+bouncycastle-api:2.25
+branch-api:2.7.0
+caffeine-api:2.9.2-29.v717aac953ff3
+checks-api:1.7.2
+cloudbees-bitbucket-branch-source:2.4.4
+cloudbees-folder:6.16
+conditional-buildstep:1.4.1
+config-file-provider:3.8.1
+configuration-as-code:1.52
+configuration-as-code-groovy:1.1
+credentials:2.6.1.1
+credentials-binding:1.27
+display-url-api:2.3.5
+docker-commons:1.18
+durable-task:493.v195aefbb0ff2
+echarts-api:5.0.2-1
+favorite:2.3.1
+font-awesome-api:5.15.2-2
+git:4.10.0
+git-client:3.9.0
+github:1.34.0
+github-api:1.114.2
+github-branch-source:2.6.0
+git-server:1.10
+google-oauth-plugin:1.0.6
+groovy:2.4
+handy-uri-templates-2-api:2.1.6-1.0
+htmlpublisher:1.25
+jackson2-api:2.13.0-230.v59243c64b0a5
+javadoc:1.0
+jenkins-design-language:1.24.8
+jira:3.7.1
+job-dsl:1.77
+jquery3-api:3.6.0-1
+jsch:0.1.55.2
+junit:1.52
+kubernetes:1.31.0
+kubernetes-client-api:5.10.1-171.vaa0774fb8c20
+kubernetes-credentials:0.9.0
+lockable-resources:2.11
+mailer:1.34
+mapdb-api:1.0.9.0
+matrix-auth:2.6.8
+matrix-project:1.19
+maven-plugin:3.7
+mercurial:2.15
+metrics:4.0.2.8
+oauth-credentials:0.4
+okhttp-api:3.12.12.2
+openshift-client:1.0.35
+openshift-login:1.0.26
+openshift-sync:1.0.53
+pam-auth:1.6
+parameterized-trigger:2.43.1
+pipeline-build-step:2.16
+pipeline-graph-analysis:1.10
+pipeline-input-step:2.12
+pipeline-milestone-step:1.3.1
+pipeline-model-api:1.9.3
+pipeline-model-definition:1.8.4
+pipeline-model-extensions:1.9.3
+pipeline-rest-api:2.15
+pipeline-stage-step:2.5
+pipeline-stage-tags-metadata:1.8.4
+pipeline-utility-steps:2.12.0
+plain-credentials:1.7
+plugin-util-api:2.4.0
+popper-api:1.16.1-2
+prometheus:2.0.10
+pubsub-light:1.13
+run-condition:1.3
+scm-api:2.6.5
+script-security:1.78
+snakeyaml-api:1.29.1
+sse-gateway:1.24
+ssh-credentials:1.19
+sshd:3.0.4
+structs:308.v852b473a2b8c
+subversion:2.15.4
+token-macro:267.vcdaea6462991
+trilead-api:1.0.13
+variant:1.4
+workflow-api:2.47
+workflow-basic-steps:2.20
+workflow-cps:2660.vb_c0412dc4e6d
+workflow-cps-global-lib:566.vd0a_a_3334a_555
+workflow-durable-task-step:2.35
+workflow-job:2.41
+workflow-multibranch:711.vdfef37cda_816
+workflow-scm-step:2.13
+workflow-step-api:622.vb_8e7c15b_c95a_
+workflow-support:3.8
```

Comment:
You changed directly the bundle but We need to change the baseplugin file then generate the bundle with make plugin-list, Is that because the PR doesnt hold this mecanism, in this case, can we add it to 4.8 too please ?

Reply:
Bundle mechanism was there earlier in 4.8, I added that to 4.8 in this PR.
That's why new file is created, and it is generated from base-plugins only.